### PR TITLE
Sync metallb 200624

### PR DESCRIPTION
**Pull request by karampok**

### CODE REVIEW

**Comment by karampok** [comment]
@fedepaol  the security check  is failing
```
✗ [High] Regular Expression Denial of Service (ReDoS) 
   Path: website/themes/hugo-theme-relearn/static/js/theme.js, line 1381 
   Info: Unsanitized user input from browser storage flows into RegExp, where it is used to build a regular expression. This may result in a Regular expression Denial of Service attack (reDOS).
```
should we try to fix the code? (or remove the website code or look if snyk can exclude that or do an issue in the theme and try to fix and then d/s). Any preference?

**Comment by fedepaol** [comment]
> @fedepaol the security check is failing
> 
> ```
> ✗ [High] Regular Expression Denial of Service (ReDoS) 
>    Path: website/themes/hugo-theme-relearn/static/js/theme.js, line 1381 
>    Info: Unsanitized user input from browser storage flows into RegExp, where it is used to build a regular expression. This may result in a Regular expression Denial of Service attack (reDOS).
> ```
> 
> should we try to fix the code? (or remove the website code or look if snyk can exclude that or do an issue in the theme and try to fix and then d/s). Any preference?

I'd just add the website to the exclusions. We don't use it anyway d/s.
At the same time it's worth having a look if there's a new version of the theme around.

File: .snyk [hunk 1]
```diff
@@ -6,3 +6,4 @@ exclude:
     - vendor/**
     - "**/*_test.go"
     - "e2etest/**"
+    - website/themes/hugo-theme-relearn/static/js/theme.js
```
Comment (fedepaol) [inline .snyk:9]: just exclude website as a whole

**Comment by karampok** [comment]
Updated, with extra commit to fix the ds dockerfile. CI should fail due to 1. Gracefull Restart (known issue), 2. FRR-K8S params (unknown why)

**Pull request merged**